src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -24,28 +25,37 @@ public Turret() {
         turret.config_kD(CONSTANTS.TURRET_PID_SLOT, CONSTANTS.D_TURRET);
         turret.config_IntegralZone(CONSTANTS.TURRET_PID_SLOT, CONSTANTS.IZONE_TURRET);
 
-        //Soft Limit
+        // Soft Limit
         turret.configForwardSoftLimitEnable(true, CONSTANTS.TURRET_ENCODER_LIMIT);
 
-
     }
 
-    public void aim(){
-        
-    }
+    // merge conflict
+    /*
+     * public void aim(double angle) { turret.set(ControlMode.Position,
+     * encoderToAngle(angle)); }
+     */
 
     public double getVel() {
         return turret.getSelectedSensorVelocity();
     }
 
-
     public void stop() {
         turret.stopMotor();
     }
 
+    public boolean found() {
+        return RobotContainer.limelight.canSeeTarget() && RobotContainer.limelight.getTargetArea() > 0.3;
```

Comment:
why do we need this? `&& RobotContainer.limelight.getTargetArea() > 0.3`

Also this shouldn't be on the turret, you can see this violates the encapsulation. I would justt remove this and below, or put them in limelight.

Reply:
I think Josiah will need to change some of the code, but I don't think that it is that much change.

Reply:
But I will do that